src/ui/EpisodesList/EpisodeItem.module.css

```diff
@@ -1,43 +1,68 @@
 .container {
   display: flex;
   align-items: center;
-  flex-direction: row;
-  padding: 0 32px;
+  column-gap: 15px;
```

Comment:
Didn't know `column-gap` works with `flex`.

Reply:
it's a new thing that is finally available in all modern browsers :D